### DISCUSSION TITLE
missing shimmer

### DISCUSSION THREAD

**Discussion by mostafahasani** [iotaledger/firefly#7807] [Help & Support]
Hi
I had some shimmer token in my old wallet but after changing of my laptop and installation of new version wallet,now my shimmer tokens are missing
please help me

**Comment by begonaalvarezd** [comment]
Quick information about SMR: what the old firefly version displayed was just a representation of what you owned, but you couldnt control the shimmers from the iota app because your rewards are in the shimmer network, not the iota network. You must download the firefly shimmer app to control those funds, you can read more about it here https://blog.shimmer.network/how-to-claim-smr-firefly/

Will close the issue because the new firefly app is not intended to display old staking rewards, but the shimmer tokens should be managed from the shimmer app. 

Please let us know if you have any other issues
